samples/mediaplayer/mediaplayer.cpp

```diff
@@ -500,7 +500,7 @@ void wxMediaPlayerApp::MacOpenFiles(const wxArrayString & fileNames )
 // ----------------------------------------------------------------------------
 
 wxMediaPlayerFrame::wxMediaPlayerFrame(const wxString& title)
-       : wxFrame(NULL, wxID_ANY, title, wxDefaultPosition, wxSize(600,600))
+       : wxFrame(NULL, wxID_ANY, title, wxDefaultPosition, wxSize(1366,768))
```

Comment:
As long as we're changing this, I'd also use `FromDIP()`. But this would require setting the size in the body, i.e. once the window is already created, so that we know which monitor it's on.